### PR TITLE
Create 1642-furthest-building-you-can-reach.js

### PR DESCRIPTION
**Pull request by aadil42**
Solved furthest-building-you-can-reach in JS.

File(s) Added: 1642-furthest-building-you-can-reach.js
Language(s) Used: JavaScript
Submission URL: https://leetcode.com/problems/furthest-building-you-can-reach/submissions/1330757405/

### CODE REVIEW

File: javascript/1642-furthest-building-you-can-reach.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ * @param {number[]} heights
+ * @param {number} bricks
+ * @param {number} ladders
+ * @return {number}
+ */
+var furthestBuilding = function(heights, bricks, ladders) {
+
+    const maxPriorityQueue = new MaxPriorityQueue({
+        compare: (a,b) => {
+            return b-a;
```
Comment (Ykhan799) [inline javascript/1642-furthest-building-you-can-reach.js:14]: Add spacing between a and b

File: javascript/1642-furthest-building-you-can-reach.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        }
+    });
+
+    let i = 0;
+
+    while (i < heights.length - 1) {
+        const diff = heights[i+1] - heights[i];
```
Comment (Ykhan799) [inline javascript/1642-furthest-building-you-can-reach.js:21]: add spacing for the i + 1 for the heights

**Comment by Ykhan799** [comment]
@aadil42 Fix the spacing for lines 13, 14, and 21. See comments above for which portions to fix

**Comment by aadil42** [comment]
@Ykhan799, fixed the spacing issue as suggested.

**Pull request merged**